cartographer/mapping/2d/tsdf_range_data_inserter_2d_test.cc

```diff
@@ -48,7 +48,7 @@ class RangeDataInserterTest2DTSDF : public ::testing::Test {
   }
 
   void InsertPoint() {
-    sensor::RangeData range_data;
+    sensor::RangeData range_data = sensor::RangeData();
```

Comment:
nit: maybe use `auto range_data = sensor::RangeData()` to remove redundancy.

Reply:
I agree, thanks.